joomla/joomla-cms
# Fixed Hard Coupling (com_users | plg_users_joomla)

Pull Request for Issue #20249

### Summary of Changes
Added `onContentPrepareForm` event to the plugin for doing the form require alterations to the passwords fields.

### Testing Instructions
Go to the administration and create a new user. Check the form that is shown need have the passwords fields with the attribute "require" if the plugin **User - Joomla!** have the Notification Mail to User option as No.

### Expected result
Passwords field with the "require" attribute 

### Actual result
Passwords field with the "require" attribute 

### Documentation Changes Required
No
